### PR TITLE
Update deployment configurations for KSP

### PR DESCRIPTION
**Pull request by esune**
Signed-off-by: Emiliano Suñé <emiliano.sune@gmail.com>

### CODE REVIEW

File: openshift/manage [hunk 1]
```diff
@@ -253,7 +261,7 @@ deployApp(){
 
     # Deploy or update the configuration ...
     updateDeploymentParams ${DEPLOYMENT_ENV_NAME}
-    if dcExists ${_appName}; then
+    if dcExists "allure-ui"; then
```
Comment (esune) [inline openshift/manage:264]: For `allure` there is no `dc` named as the app label, so I picked one of the two components to check whether the app was already deployed.

**Comment by esune** [comment]
Please note that due to the nature of the service, all traffic between containers is via Internet so the only policies that are necessary are the ones to enable Ingress traffic to the pod(s).

File: openshift/templates/nsp/build.yaml [hunk 2]
```diff
@@ -4,59 +4,65 @@ apiVersion: v1
...
-      name: pods-to-k8s-api
-      labels:
-        name: pods-to-k8s-api
-        env: ${TAG_NAME}
+      name: any-to-any
     spec:
       description: |
-        Allow pods to talk to the internal k8s api so deployments work.
-        This only needs to be specified once per environment.
+        Disable Aporeto policies: Allow all pods within the namespace to communicate.
```
Comment (WadeBarnes) [inline openshift/templates/nsp/build.yaml]: `:` in the description will cause the KNP tp be invalid.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
See comment

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**